charts/external-dns/templates/deployment.yaml

```diff
@@ -132,6 +136,47 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+        {{- if eq $provider "webhook" }}
+        {{- with .Values.provider }}
+        - name: {{ $providername }}
+          image: {{ get $whp .name }}
+          imagePullPolicy: {{ $.Values.image.pullPolicy }}
+          {{- with .env }}
+          env:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .args }}
+          args:
+            - {{ toYaml . | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: http
+              protocol: TCP
+              containerPort: 8888
+          livenessProbe:
+            {{- toYaml $.Values.livenessProbe | nindent 12 }}
+          readinessProbe:
+            {{- toYaml $.Values.readinessProbe | nindent 12 }}
+          {{- if or $.Values.secretConfiguration.enabled $.Values.extraVolumeMounts }}
+          volumeMounts:
+            {{- if $.Values.secretConfiguration.enabled }}
```

Comment:
These should test `.mountPath`, not `.enabled`.

Reply:
That's not the subject of this PR.
Maybe it can be done, I don't know. 
For removing `.enabled` on volumes, I suggest to open an other PR, to stay focus on webhook provider for this PR.